Makefile

```diff
@@ -65,6 +65,9 @@ build: fmt
 	@cp -ru $(CURPATH)/vendor/* $(TARGET_DIR)/src
 	@GOPATH=$(BUILD_GOPATH) $(GOBUILD) $(LDFLAGS) -o $(TARGET) $(MAIN_PKG)
 
+build-fast:
```

Comment:
side note: i think we need to consolidate our `make` targets like @alanconway has suggested
this isn't a blocker but maybe we skip this target for now and circle back on all our current targets as tech debt?

Reply:
@ewolinetz @alanconway This one is only for a local build. The problem with the `build` target is that it copies all the sources, causing Goland IDE to reindex sources.